src/features/Navigation/Navigation.tsx

```diff
@@ -49,8 +51,13 @@ const NavigationBar = () => {
         <NavigationItems items={navigationItems} />
         <Dropdown items={dropDownItems} />
         <NavLink id="logout-link" to="/logout" className="navbar-link">
-          <div>Kirjaudu ulos</div>
+          <div>{t('logoutButton')}</div>
```

Comment:
Please type the t-function using this doc as a reference https://react.i18next.com/latest/typescript =)

Reply:
Type declaration added!

Reply:
Need to fix two things to make it work,

the declaration should be to module i18next instead of react-i18next:

![image](https://user-images.githubusercontent.com/34128180/208909989-30f904cf-2b02-475e-9f5b-04352c767b0c.png)

Then in tsconfig we need to include it

![image](https://user-images.githubusercontent.com/34128180/208910110-77ecc714-57dc-4760-bb2d-6ea01c0abd58.png)


But I would also love to see the file renamed to i18next.d.ts
And because we have already two type declarations inside source folder, it would be better to move them in a @types folder in same level as source-folder. Its a convention. =P

Reply:
Then it works and can test it: 

![image](https://user-images.githubusercontent.com/34128180/208910424-d26c4110-88da-4c7f-b611-1c65c679dc51.png)

Reply:
But you can fix this easily also in the next PR too - i can just approve to unblock you.

Reply:
Thanks for the corrections! I'll merge this and fix them in the next PR!